crates/rspack_macros/Cargo.toml

```diff
@@ -11,5 +11,13 @@ proc-macro = true
 
 [dependencies]
 proc-macro2 = { workspace = true }
-quote       = { workspace = true }
-syn         = { workspace = true, features = ["full", "visit-mut"] }
+quote = { workspace = true }
+syn = { workspace = true, features = [
+  "full",
```

Comment:
？？？ why appears in this script PR ?

Reply:
This may be because we configured `default-features = false` for all dependencies.